ArrtSource/App/AppWindow.cpp

```diff
@@ -129,37 +128,35 @@ ArrtAppWindow::ArrtAppWindow()
 
     StorageBrowser->SetStorageAccount(m_storageAccount.get(), StorageEntry::Type::Other, m_lastStorageDisplayContainer, QString());
 
-    Viewport->SetSceneState(m_sceneState.get());
+    RenderingTab->Viewport->SetSceneState(m_sceneState.get());
```

Comment:
I've moved some UI elements into a sub widget, which fixes a tab order issue and separates some config stuff out into its own file, which is generally easier to work with.